kuma/static/js/font-check.js

```diff
@@ -36,7 +36,9 @@ var mdn = window.mdn || {};
                 // skip the FFO loading routine
                 fonts[i].loaded = true;
             }
-        } catch(e) {}
+        } catch(e) {
+            console.error('Font-Check :: Error while reading from sessionStorage', e);
```

Comment:
@stephaniehobson Note that the warning here and in `fonts.js` below will be resolved by https://github.com/mozilla/kuma/pull/4545/files#diff-e4403a877d80de653400d88d85e4801aR19

Reply:
👍